src/commands/entities/rename.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2022, Nuance, Inc. and its contributors.
+ * All rights reserved.
+ *
+ * This source code is licensed under the Apache-2.0 license found in
+ * the LICENSE file in the root directory of this source tree.
+ */
+
+import {flags} from '@oclif/command'
+import makeDebug from 'debug'
+
+import * as EntitiesAPI from '../../mix/api/entities'
+import * as MixFlags from '../../utils/flags'
+import MixCommand from '../../utils/base/mix-command'
+import {EntitiesRenameParams, MixClient, MixResponse, MixResult} from '../../mix/types'
+import {DomainOption} from '../../utils/validations'
+import chalk from 'chalk'
+
+const debug = makeDebug('mix:commands:entities:rename')
+
+export default class EntitiesRename extends MixCommand {
+  static description = `rename an entity
+
+Use this command to rename an entity in a project.`
+
+  static examples = [
+    '$ mix entities:rename -P 1922 -E DrinkSize --name DrinkFormat',
+  ]
+
+  static flags = {
+    entity: MixFlags.entityFlag,
+    json: MixFlags.jsonFlag,
+    name: MixFlags.entityNameFlag,
```

Comment:
For `channels:rename`, I went for `--new-name` for this flag. What's your preference between the two?

Reply:
I went with `--name` because that's what `projects:rename` uses. I agree it's trickier with entites since the entity ID is also its name. But I woudl stick to `--name` to remain consistent with `projects:rename`.